src/librustc/mir/visit.rs

```diff
@@ -519,12 +519,12 @@ macro_rules! make_mir_visitor {
                         resume_arg,
                         drop: _,
                     } => {
+                        self.visit_operand(value, source_location);
```

Comment:
Where does this reordering matter?

Reply:
It doesn't seem to matter from what I've seen, but it aligns the order of `Yield` with the one for `Call`. That seems less surprising to me.